Assets/Scripts/Scriptables/Systems/DamageResistanceSystem.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Celeritas.Game;
+using Celeritas.Game.Entities;
+using Celeritas.Scriptables.Interfaces;
+using Sirenix.OdinInspector;
+using UnityEngine;
+
+namespace Celeritas.Scriptables.Systems
+{
+
+	/// <summary>
+	/// Reduces incoming damage by set percentage from all sources.
+	/// </summary>
+
+	[CreateAssetMenu(fileName = "New Ship Damage Resistance Modifier", menuName = "Celeritas/Modifiers/Ship Damage Resistance")]
+
+	public class DamageResistanceSystem : ModifierSystem, IEntityEffectAdded, IEntityEffectRemoved
+	{
+
+		[SerializeField, PropertyRange(-100, 100), Title("Damage Taken Modifier Percentage")]
```

Comment:
Don't need to change anything, but if you want to add little notes to things, you can go Title("title","subtitle"). Eg a handy subtitle here might be that - percentages reduce damage. Just in case you didn't know; I've found it handy c:

Reply:
Good idea, I've added it in.